datacenter/ucp/2.1/guides/admin/upgrade/release-notes.md

```diff
@@ -10,6 +10,59 @@ known issues for the latest UCP version.
 You can then use [the upgrade instructions](index.md), to
 upgrade your installation to the latest release.
 
+
+## Version 2.1.1
+
+(13 Mar 2017)
+
+**New features**
+
+* Core
+  * Administrators can now configure the frequency with which UCP polls metrics.
+  Use `docker service update --env-add METRICS_SCRAPE_INTERVAL=2h ucp-agent`,
```

Comment:
Lets set this to something like `10m` for the example.  Using 2h might be confusing with the storage interval